ERCS/erc-x.md

```diff
@@ -0,0 +1,102 @@
+---
+eip: x
+title: Simple Permissions Checks
+description: Standardized interface for checking either contract-level or calldata-level permissions
+author: TODO
+discussions-to: TODO
```

Comment:
Please create a discussions topic in Eth Magicians with a link to this PR.
https://ethereum-magicians.org/